sections/image-banner.liquid

```diff
@@ -313,17 +313,21 @@
           "type": "select",
           "id": "heading_size",
           "options": [
+            {
+              "value": "small",
+              "label": "t:sections.all.heading_size.options__1.label"
+            },
             {
               "value": "medium",
-              "label": "t:sections.image-banner.blocks.heading.settings.heading_size.options__1.label"
+              "label": "t:sections.all.heading_size.options__2.label"
             },
             {
               "value": "large",
-              "label": "t:sections.image-banner.blocks.heading.settings.heading_size.options__2.label"
+              "label": "t:sections.all.heading_size.options__3.label"
             }
           ],
           "default": "medium",
```

Comment:
Just tested and got the same effect.

Reply:
Question for @martinamarien - I see that the rest of the implementations are using `{{ block.settings.title_size }}` with the value being the heading class like `h1`, etc...

I'm assuming that the reason you didn't do that here is to prevent unwanted changes for merchants who are upgrading.  I'm wondering if the technical debt of having a slightly different implementation here is worth the better upgrade experience if we can include it in the release notes that they'd need to check this.

I guess it's not a huge difference implementation-wise, but I'd love for it to be consistent across the board.  The more exceptions we make during these parity exercises due to previous implementations, the more each setting will have two or more different implementations.

Reply:
When I am adding a new section, it's defaulting to medium for me!

Reply:
> I'm assuming that the reason you didn't do that here is to prevent unwanted changes for merchants who are upgrading. I'm wondering if the technical debt of having a slightly different implementation here is worth the better upgrade experience if we can include it in the release notes that they'd need to check this.

This is exactly the reason I didn't convert it to h0/h1/h2. Since we're already introducing a change to the default heading value (breaking), I'm happy to update Image Banner and Slideshow to use the classes instead. We just need to clearly communicate this to merchants in the release notes, like you said.